packages/@react-native-windows/tester/package.json

```diff
@@ -17,7 +17,8 @@
     "@react-native-picker/picker": "2.2.0",
     "react": "17.0.2",
     "react-native": "0.0.0-20211021-2008-eccbf9b5c",
-    "react-native-windows": "^0.0.0-canary.414"
+    "react-native-windows": "^0.0.0-canary.414",
+    "react-native-xaml": "^0.0.50"
```

Comment:
Are any APIs changing? If so should we pin the version here?

Reply:
Esp if we are snapshotting the output of its components.

Reply:
apis are coming from xaml so they should be stable

Reply:
But are `react-native-xaml` projections stable? E.g. I remember a change from strings to enums before, that would be breaking.